### ISSUE TITLE
[Feature Request] Predictions with GPLVM

### ISSUE THREAD

**Issue by lion241** [cornellius-gp/gpytorch#1966]
# 🚀 Feature Request

Predictions with GPLVM and feature representation.

https://docs.gpytorch.ai/en/stable/examples/045_GPLVM/Gaussian_Process_Latent_Variable_Models_with_Stochastic_Variational_Inference.html

## Motivation

**Is your feature request related to a problem? Please describe.**

The best thing about the GP is that the covariance matrix can be useful as logits for a classifier or something else since it tells me how things are related and unlike a neural network I can pass the entire distribution as input.

## Pitch

**Describe the solution you'd like**

Consider I have training data with input and X and output Y  . I can group X and Y over the training data (the input to the GPLVM would be (X,Y)) and for testing do Gaussian conditioning to perform a prediction on Y given a test point X, by that I mean Y(test)|X(test),Z (test).

It would be interested if I could extract the respective mean and covariance of Y(test)|X(test),Z  as well because i could use it for a downstream task. 

**Describe alternatives you've considered**

I don't see any way I can input to GPLVM model the test point X to get a conditional Gaussian for the test point Y.

## Additional context
I can easily do the gaussian conditioning since its just a formula but I still need a way give an input to the model.
